.github/workflows/wheel_tests_and_release.yml

```diff
@@ -53,24 +51,23 @@ jobs:
         env:
           CIBW_BUILD: ${{ matrix.cibw_python }}-*
           CIBW_ARCHS_LINUX: ${{ matrix.cibw_arch }}
-          CIBW_SKIP: "*-musllinux_*"
-          CIBW_MANYLINUX_X86_64_IMAGE: ${{ matrix.cibw_manylinux }}
-          CIBW_MANYLINUX_I686_IMAGE: "manylinux2010"
+          CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
+          CIBW_MANYLINUX_I686_IMAGE: manylinux2014
+          CIBW_MUSLLINUX_X86_64_IMAGE: musllinux_1_1
```

Comment:
Note: here I bumped up the i686 image from `manylinux2010` to `manylinux2014`. If we want to bump to musllinux_1_2 instead of musllinux_1_1, please let me know.

Reply:
Unless I am missing something, `musllinux_1_1` will have broader compatibility, and it's not costing us performance or anything else to build against Musl 1.1. So better stay with `1_1`?

Reply:
Sure, thanks for the clarification!